library/std.pn

```diff
@@ -4,4 +4,5 @@ finite_num(x) := "F(G(!x))"
 zero(x) := "G(!x)"
 
 #context("adder", "bin_add")
+#context("compare", "bin_less")
```

Comment:
Maybe we can call this "less" or something instead of "compare" (compare less clear as to which way the comparison works).

Reply:
Changed in new commit.